phoenicis-library/src/main/java/org/phoenicis/library/ShortcutManager.java

```diff
@@ -120,7 +120,7 @@ public void createShortcut(ShortcutDTO shortcutDTO) {
     public void uninstallFromShortcut(ShortcutDTO shortcutDTO, Consumer<Exception> errorCallback) {
         final InteractiveScriptSession interactiveScriptSession = scriptInterpreter.createInteractiveSession();
 
-        interactiveScriptSession.eval("include(\"engines.wine.shortcuts.reader\");",
+        interactiveScriptSession.eval("const ShortcutReader = include(\"engines.wine.shortcuts.reader\");",
```

Comment:
Not really related to this PR but: do we have the engine ID in the places where currently "wine" is hard coded?

Reply:
This is a good point. Sadly no, we don't seem to have the engine id at hand here. I think we will need to add it to the `ShortcutDTO` class to have it available. 

What do you think?

Reply:
Seems reasonable. Can you describe the idea in an issue?